x-pack/plugins/secops/public/pages/hosts/host_details.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// @ts-ignore: EuiBreadcrumbs has no exported member
+import { EuiBreadcrumbs, EuiPanel, EuiTitle } from '@elastic/eui';
+import React from 'react';
+import { pure } from 'recompose';
+import styled from 'styled-components';
+import { getHostsUrl, HostComponentProps } from '../../components/link_to/redirect_to_hosts';
+
+export const HostDetails = pure<HostComponentProps>(({ match }) => (
+  <div>
```

Comment:
nit: If / when you don't truly need a `<div>`, you can use the [short fragment syntax](https://reactjs.org/docs/fragments.html#short-syntax): `<>`.  No change requested, just FYI / general feedback.

Reply:
I highly recommend the `<>` when and where possible as more than a nit. The DOM these days fills up rapidly with tags and starts hurting performance and anything we can do to keep minimize tags is helpful.